### ISSUE TITLE
QDS not accepting liquids/counting quest progress (1.7.10)

### ISSUE THREAD

**Issue by Cynnimon** [lorddusk/HQM#390]
Players on server are complaining that the QDS is not accepting liquid from various sources when attempting to complete quests. Note, this is a 1.7.10 pack (Divine Journey).

This issue continues on in single player testing as well, when attempting to draw liquid from a source (in this case drum) using a transfer node (liquid). The liquid drains, quest progress stays stuck at 0%.

Any help would be appreciated!

**Comment by canitzp** [comment]
Is the liquid actually drained at all, so is the drum getting emptier? I only can imagine that there is a issue with the transfer node and if it is a issue with HQM, then I'm sorry to tell that 1.7.10 doesn'T get a update anytime soon

**Comment by Cynnimon** [comment]
Canitzp - Yes the liquid drains as if it is being consumed, but the quest progress remains at 0%, no matter how much liquid is pumped into it. I tested it to see if it was the transfer node per your suggestion, and instead used a fluid conduit and a pressurized fluid conduit, and the same issue occurred. The barrel drained, the pipe had fluid moving through it, but the QDS remained at 0%. :)

**Comment by canitzp** [comment]
How have you setuped the quest? which type of quest and which item did you used as aim?

**Comment by Cynnimon** [comment]
I did aim the quest, in particular I'm testing on the QDS accepting gelid cryotheum for the cryo world creative quest. I gave it both a drum and a bedrockium drum to test from. They are aimed to the QDS, I attempted all 6 faces, and 3 different means of piping it in (Transfer node liquid, ender fluid conduit, ender pressurised conduit) and still the QDS remains at 0%. Players on our server have to have all the quests manually checked off to progress. Hope this helps :)
